src/Avalonia.Controls/Slider.cs

```diff
@@ -181,6 +181,26 @@ public TickPlacement TickPlacement
             set { SetValue(TickPlacementProperty, value); }
         }
 
+        /// <summary>
+        /// Gets a value indicating whether the <see cref="Slider"/> is currently being dragged.
+        /// </summary>
+        protected bool IsDragging => _isDragging;
+
+        /// <summary>
+        /// Gets the <see cref="Track"/> part of the <see cref="Slider"/>.
+        /// </summary>
+        protected Track? Track => _track;
+
+        /// <summary>
+        /// Gets the <see cref="Button"/> that decreases the <see cref="Slider"/> value.
+        /// </summary>
+        protected Button? DecreaseButton => _decreaseButton;
```

Comment:
The buttons probably don't need to be exposed. They aren't used anywhere if I recall correctly. I made then protected fields in case derived controls ever need them (not considering it would be frowned upon from an API standpoint).

Reply:
Yeah I noticed they weren't used, but assumed that they were exposed for a reason. I can remove these then.

Reply:
Yea, sorry, there was no reason other than I thought it might be useful for derived controls in the future (since I already did need Track).

Reply:
Done!